scripts/exrrfs_post.sh

```diff
@@ -433,6 +436,10 @@ fi
 #
 cpreq -p ${prslev} ${COMOUT}
 cpreq -p ${natlev} ${COMOUT}
+# Only one latlons_corners file per cycle is needed in COMOUT - make this change later
+if [ ${PREDEF_GRID_NAME} = "RRFS_FIREWX_1.5km" ]; then
+  cpreq -p latlons_corners.txt.f${fhr} ${COMOUT}
+fi
```

Comment:
Only one latlons_corners text file is needed in COMOUT - this can be changed later.